app.yaml

```diff
@@ -10,3 +10,8 @@ handlers:
   static_files: build/index.html
   upload: build/index.html
   secure: always
+  http_headers:
+    X-Frame-Options: "DENY"
```

Comment:
Wouldn't we also want to add `Content-Security-Policy: frame-ancestors 'none'`?
https://infosec.mozilla.org/guidelines/web_security#x-frame-options

Reply:
`X-Frame-Options: deny` takes care of what `Content-Security-Policy: frame-ancestors 'none'`, however we must look into setting up a more robust CSP, and I'll follow up on that as I want to make sure we don't break anything.